### PR TITLE
docs: remove duplicate buckets

### PR DESCRIPTION
**Pull request by JStickler**
**What this PR does / why we need it**:

Fixes issue reported via docs @ grafana email.

completes backport of #12641 that was started in #13025 but missed this part of the content.

### CODE REVIEW

File: docs/sources/setup/install/helm/install-monolithic/_index.md [hunk 1]
```diff
@@ -81,9 +81,6 @@ If you set the `singleBinary.replicas` value to 2 or more, this chart configures
             schema: v13
         storage:
           bucketNames:
```
Comment (ddaskal) [inline docs/sources/setup/install/helm/install-monolithic/_index.md:83]: You should also remove line 83 so we don't have two `bucketNames` keys

**Review by ddaskal — CHANGES_REQUESTED** [review]

File: docs/sources/setup/install/helm/install-monolithic/_index.md (hunk 1 above)
Comment (JStickler) [reply docs/sources/setup/install/helm/install-monolithic/_index.md:83]: Ah, good catch, thanks!

**Review by trevorwhitney — CHANGES_REQUESTED** [review]
why are we removing these? these are required for helm to work in any cloud environment.

**Review by trevorwhitney — APPROVED** [review]
nevermind, I see from the other context these are duplicated. lgtm

**Pull request merged**